test/clientAuth-bearerBody-test.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('chai').assert;
+const AccessTokenRequest =
+    require('../oicMsg/oauth2/init.js').AccessTokenRequest;
+const AccessTokenResponse =
+    require('../oicMsg/oauth2/init.js').AccessTokenResponse;
+const AuthorizationRequest =
+    require('../oicMsg/oauth2/init.js').AuthorizationRequest;
+const AuthorizationResponse =
+    require('../oicMsg/oauth2/init.js').AuthorizationResponse;
+const BearerBody = require('../src/clientAuth/bearerBody').BearerBody;
+const Client = require('../src/oauth2/init').Client;
+const CLIENT_AUTHN_METHOD =
+    require('../src/clientAuth/clientAuth').CLIENT_AUTHN_METHOD;
+const ResourceRequest = require('../oicMsg/oauth2/init.js').ResourceRequest;
+
+const CLIENT_ID = 'A';
+
+const CLIENT_CONF = {
+  'issuer': 'https://example.com/as',
```

Comment:
You don't need the quote around the keys here, remove here and below.

Reply:
Fixed